tests/inside_worker_test/cast_to_float_or_null_test.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+import sqlalchemy
+
+from tests.inside_worker_test.conftest import slow
+
+
+@pytest.fixture(params=[2, 2.2, 3.898986, "3.898986", "6", "0.2", 0.6])
+def valid_float_representation(request):
+    return request.param
+
+
+@pytest.fixture(params=["a2", "10b", "3.898986k", "3k.898986", "l6.9"])
```

Comment:
The last param is easily misread for `16.9`, so that makes this fixture a bit confusing.

Might also be good to have examples with whitespace or with decimal separators of other locales.